backend/webapi/Features/Parties/ProfileStatus.cs

```diff
@@ -134,6 +138,7 @@ public class LicenceDeclarationDto
         public string? Email { get; set; }
         public string? Phone { get; set; }
         public string? Cpn { get; set; }
+        public IEnumerable<CredentialDto> Credentials { get; set; } = Enumerable.Empty<CredentialDto>();
```

Comment:
I think a cleaner approach would be to have a single boolean property of `HasBCProviderCredential` that can be mapped in the mapping profile.